packages/linux/package.mk

```diff
@@ -16,6 +16,12 @@ PKG_STAMP="$KERNEL_TARGET $KERNEL_MAKE_EXTRACMD"
 PKG_PATCH_DIRS="$LINUX"
 
 case "$LINUX" in
+  amlogic-5.2)
```

Comment:
```suggestion
  amlogic)
```

Reply:
I'm already on 5.3-pre-rc1 so for me 5.2 is stoneage :) .. and because so many different kernels are being used I prefer to keep the numbering obvious until we drop the separate kernel tree and use upstream (5.4 earliest).

Reply:
oki no problem :)